st2common/st2common/util/virtualenvs.py

```diff
@@ -169,7 +169,7 @@ def create_virtualenv(virtualenv_path, logger=None, include_pip=True, include_se
     logger.debug('Creating virtualenv in "%s" using Python binary "%s"' %
                  (virtualenv_path, python_binary))
 
-    cmd = [virtualenv_binary]
+    cmd = [virtualenv_binary, '-v']
```

Comment:
It's odd that you have to modify StackStorm's internals just to get it to run properly on Travis. It just seems like that is "fixing" the problem from the wrong end.

Instead of making StackStorm run better on Travis (which implies that it _doesn't_ run as well on non-Travis machines, like users' machines, which should be a warning flag!), I think the ultimate solution will be correcting the Travis configuration to properly run StackStorm.

It's best to have your test environment resemble a normal production environment as much as humanly possible. Otherwise you end up with software that runs tests well but doesn't actually run user workloads well.

Reply:
This was just for increasing the debugging output as well.